src/ui/src/components/core/base/BaseInputRange.vue

```diff
@@ -65,8 +65,10 @@ const thumbRadius = 9;
 const thumb = ref<HTMLElement>();
 const slider = ref<HTMLElement>();
 
-const precision = computed(() => Math.ceil(-Math.log10(props.step)));
-const displayValue = computed(() => model.value.toFixed(precision.value));
+const precision = computed(() => String(props.step).split(".")[1]?.length ?? 0);
+const displayValue = computed(() =>
+	Number(model.value).toFixed(precision.value),
+);
 
 const progress = computed(() => {
```

Comment:
regarding the behavior of model, I think we should also write this change

```
const progress = computed(() => {
	const v =
		typeof model.value !== "number" ? Number(model.value) : model.value;

	return ((v - props.min) / (props.max - props.min)) * 100;
});
```

Reply:
It's a good point, the thing is the empty state is always a `string` because of this

https://github.com/writer/writer-framework/blob/9f4af7b7f5c5bc247bc8174a900d6c98456ef529/src/ui/src/renderer/useFormValueBroker.ts#L113

Which make things a bit weird to handle... That's why I choose to put the thumb in the middle.

In fact, I can't display `0` neither because the `min` could be greater. Thus, I think the best fix is to display nothing in this situation like this :

```ts
const displayValue = computed(() => {
	if (typeof model.value !== "number") return "";
	return Number(model.value).toFixed(precision.value);
});
```

I updated it, well spoted

Reply:
I feel the experience is not great for the user.

![image](https://github.com/user-attachments/assets/0aefb0e4-2439-4fa8-814f-61daf599affa)

and something like that ? It will render

![image](https://github.com/user-attachments/assets/771ba407-9a76-4223-a30a-9ede2369ddd2)

```typescript
const displayValue = computed(() => {
	const v = model.value !== "" ? model.value : props.min;
	return Number(v).toFixed(precision.value);
});

const progress = computed(() => {
	const v = model.value !== "" ? model.value : props.min;
	return ((v - props.min) / (props.max - props.min)) * 100;
});
```